src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -0,0 +1,85 @@
+import { arrowKeysNavigation, getRowAndColumnCount } from '../handleKeyPress';
+
+const { JSDOM } = require('jsdom');
+
+describe('handle-key-press', () => {
+  describe('moveToNextFocus', () => {});
+
+  describe('arrowKeysNavigation', () => {
+    const evt = {};
+    const rowAndColumnCount = {};
+    let rowIndex;
+    let colIndex;
+
+    it('should stay the current cell', () => {
```

Comment:
Technically this can happen for all directions. If you want to you can add three more tests for this, but this is fine